Add lockup end date alongside weeks count

tomhirst opened the pull request:
Addresses #61

Replacing the large week counter figure with a date string didn't look right, so I added the end date underneath in addition. This approach allows the alignment of the figures and their unit types to remain uniform, too.

This will show for any existing lockup (not just those of < 1 week) as it's useful to know the exact date no matter how many weeks your tokens are locked up for.

![Screenshot 2022-04-26 at 12 16 31](https://user-images.githubusercontent.com/2827412/165288348-0aa1ec23-53bb-4a1f-ac0c-b99ee931c7f2.png)

joshfraser commented:
Hmm, I don't love how this looks. What if the large text said "0 weeks" and we replaced the "Weeks" label with the date?

Nit, but let's add a comma between the date and the year.

tomhirst commented:
> Hmm, I don't love how this looks. What if the large text said "0 weeks" and we replaced the "Weeks" label with the date?
> 
> Nit, but let's add a comma between the date and the year.

Makes sense! Implemented:

![Screenshot 2022-04-27 at 12 17 38](https://user-images.githubusercontent.com/2827412/165507274-3c5de731-44a5-4967-83d8-dfe695521e03.png)
